Source/Charts/Renderers/ChartDataRendererBase.swift

```diff
@@ -55,7 +55,9 @@ open class DataRenderer: Renderer
     {
         guard let data = dataProvider?.data
             else { return false }
-        
-        return data.entryCount < Int(CGFloat(dataProvider?.maxVisibleCount ?? 0) * (viewPortHandler?.scaleX ?? 1.0))
+
+        let maxVisibleCount = CGFloat(dataProvider?.maxVisibleCount ?? 0)
+        let scale = self.viewPortHandler?.scaleX ?? 1.0
```

Comment:
I got `self` is improving the speed. But can we just uses self, without using two instants? like
```swift
return data.entryCount < Int(CGFloat(dataProvider?.maxVisibleCount ?? 0) * (self.viewPortHandler?.scaleX ?? 1.0))
```
I assume it has the same effect?